src/tests/MetadataMediatorTest.cpp

```diff
@@ -236,8 +237,8 @@ void MetadataMediatorTest::testDataset0() {
   // verify set to values in dataset_config file
   newRenderAssetHandle = objAttr->getRenderAssetHandle();
   // verify same renderasset handle to loaded stage attributes
-  CORRADE_VERIFY(newRenderAssetHandle.find("dataset_test_object3.glb") !=
-                 std::string::npos);
+  CORRADE_COMPARE_AS(newRenderAssetHandle.find("dataset_test_object3.glb"),
+                     std::string::npos, Cr::TestSuite::Compare::NotEqual);
```

Comment:
I'm still figuring out the semantics and desired usage of this as I go, but I think when `expected` is a known value instead of a variable (such as the `npos` here), `NotEqual` will not have any added value compared to a plain `!=`, because you already know what value it shouldn't be.

Same goes with `-1` / `ID_UNDEFINED` elsewhere, I also don't think `NotEqual` is useful in those cases.

(That said, some `StringContains` comparator, which *would* provide better diagnostics, is on my TODO list. But not today :))